pkg/pgsql/server/stmts_handler.go

```diff
@@ -0,0 +1,36 @@
+package server
+
+import (
+	"regexp"
+)
+
+func (s *session) isInBlackList(statement string) bool {
+	var set = regexp.MustCompile(`(?i)set\s+.+`)
```

Comment:
Compiling regex may be a bit heavy operation if done on each statement -  can we move this to global var?